internal/webservice/helpers.go

```diff
@@ -95,6 +146,16 @@ func parseHistoryFilters(r *http.Request) (*repository.HistoryFilters, error) {
 	return &filters, nil
 }
 
+func getClusterIDQueryParam(r *http.Request) string {
```

Comment:
Since we want to set nil if value is not present, why don't we return `*string`. If the value is empty, return nil. otherwise, return pointer to the query string. For integer types, if value is empty, return nil. Otherwise, try to parse it. Then return the value of the parsed string or error

Reply:
Updated